src/main/menu_app/src/App.js

```diff
@@ -12,15 +12,16 @@ import './App.css';
 const App = () => {
   return (
     <div className="app">
-      <h1>Menu Generator</h1>
-        <Router>
-          <Navbar className='navbar' />
+      <Router>
+        <Navbar />
+        <div className="container-fluid">
```

Comment:
It looks like this commit message is slightly misleading now? How are you adding bootstrap files here? By this `className`? Much more is happening here with router and navbar, no?

Reply:
* It said bootstrap styles.